units/systemd-nspawn@.service.in

```diff
@@ -13,7 +13,7 @@ Before=machines.target
 After=network.target
 
 [Service]
-ExecStart=@bindir@/systemd-nspawn --quiet --keep-unit --boot --link-journal=try-guest --network-veth --settings=override --machine=%i
+ExecStart=@bindir@/systemd-nspawn --quiet --keep-unit --boot --link-journal=try-guest --network-veth -U --settings=override --machine=%i
```

Comment:
As per https://github.com/systemd/systemd/pull/3093/commits/f845969d4b19843efb27b1a007ea97e2416e3e57#diff-db6b16c9ade7e951e3bb793bec110098R215 , `-U` is the same as `--private-users=auto` . But the code does not appear the same.

I'd say that having a short option without corresponding long option version would be very weird. What about `--private-users=try`? That would convey that the usage depends on the host system's ability to use user namespaces.